clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/ClientConfig.java

```diff
@@ -60,6 +60,13 @@
   private boolean isHttps = false;
   private SSLFactory sslFactory = null;
 
+  // HttpTransport settings
+  private int maxConnectionsPerRoute = 2; // AsyncHttpClient Default, only for HTTP1
+
+  private int maxConnectionsTotal = 20; // AsyncHttpClient Default, only for HTTP1
+
```

Comment:
So Venice Thin Client doesn't AsyncHttpClient any more, right?
I guess you can remove `AsyncHttpClient` from the javadoc.